dbt/adapters/databricks/python_submissions.py

```diff
@@ -414,9 +429,11 @@ def get_elapsed() -> float:
         while get_elapsed() < MAX_CLUSTER_START_TIME:
             status_response = self.get_cluster_status()
             if str(status_response.get("state")).lower() == "running":
-                return
-            else:
-                time.sleep(5)
+                libraries_status_response = self.get_cluster_libraries_status()
```

Comment:
I'm surprised this isn't done as part of 'STARTING'.  Does every run need to wait for library installs, or just those that specify additional libraries?

Reply:
Good question. I _assume_ that basic default libraries are a part of 'STARTING' indeed and only the libraries that are added explicitly (the ones exposed via Libraries API) are not and need extra time so that python models do work correctly.

Reply:
If that's the case, let's take this branch only if additional libraries have been specified.